### PR TITLE
Added Security and Infra

### PR DESCRIPTION
**Pull request by bitwiseman**
Per discussion on IRC

### CODE REVIEW

File: jep/1/README.adoc [hunk 1]
```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
...
-  All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the sponsor proposes to deal with these incompatibilities.
-  If there are no dackwards compatibility concerns, the section must say that.
-  JEP submissions that do not adequately discuss backwards compatibility,
-  when such discussion is required will not be approved as JEP Drafts.
+. **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
+  The JEP must explain how the it proposes to deal with these incompatibilities.
+  If there are no backwards compatibility concerns, the section must say that.
+. **Security** - All JEPs must include a section describing any security risks and their severity.
+  The JEP must explain how the it proposes to deal with these risks.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: Grammar: how the it proposes

File: jep/1/README.adoc [hunk 2]
```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
...
 during discussion.
 
-. **Backwards Compatibility** -
-  All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the sponsor proposes to deal with these incompatibilities.
-  If there are no dackwards compatibility concerns, the section must say that.
-  JEP submissions that do not adequately discuss backwards compatibility,
-  when such discussion is required will not be approved as JEP Drafts.
+. **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
+  The JEP must explain how the it proposes to deal with these incompatibilities.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: Grammar: how the it proposes

File: jep/1/README.adoc [hunk 3]
```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
...
+  If there are no backwards compatibility concerns, the section must say that.
+. **Security** - All JEPs must include a section describing any security risks and their severity.
+  The JEP must explain how the it proposes to deal with these risks.
+  If there are no security concerns, the section must say that.
+. **Infrastructure Requirements** -
+  All JEPs must include a section describing any infrastructure changes or addtions they require.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: typo: addtions

File: jep/1/README.adoc [hunk 4]
```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
...
-. **Backwards Compatibility** -
-  All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the sponsor proposes to deal with these incompatibilities.
-  If there are no dackwards compatibility concerns, the section must say that.
-  JEP submissions that do not adequately discuss backwards compatibility,
-  when such discussion is required will not be approved as JEP Drafts.
+. **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
+  The JEP must explain how the it proposes to deal with these incompatibilities.
+  If there are no backwards compatibility concerns, the section must say that.
+. **Security** - All JEPs must include a section describing any security risks and their severity.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: more generally:

… describing its impact on the security of affected components, and, if any, how these are dealt with. This includes discussion on potential security issues and how they are prevented, as well as how the proposed changes interact with existing permissions, authentication, authorization, etc.

File: jep/1/README.adoc (hunk 1 above)
Comment (daniel-beck) [inline jep/1/README.adoc]: Obsolete if my proposed phrasing is picked up.

File: jep/1/README.adoc [hunk 5]
```diff
@@ -552,12 +552,16 @@ The reasoning should provide evidence of consensus within the
...
+  The JEP must explain how the it proposes to deal with these risks.
+  If there are no security concerns, the section must say that.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: s/concerns/impact perhaps (and rephrase as needed)?

File: jep/1/README.adoc (hunk 3 above)
Comment (daniel-beck) [inline jep/1/README.adoc]: As the security section this is too narrow. E.g. (and IIRC) @rtyler was annoyed by nobody discussing with him impact of the setup wizard before we rolled out 2.0 and it hammered the update sites.

So, more generally, *impact on* Jenkins project infrastructure. (Adapt the next few lines as needed.)

**Comment by bitwiseman** [comment]
@oleg-nenashev 
If it is important to you that we add a "Testing" section, please file a PR as noted in #13, and people can discuss there.

File: jep/1/README.adoc [hunk 6]
```diff
@@ -553,15 +553,22 @@ community and discuss important objections or concerns raised
 during discussion.
 
 . **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the it proposes to deal with these incompatibilities.
+  The JEP must explain how it proposes to deal with these incompatibilities.
   If there are no backwards compatibility concerns, the section must say that.
-. **Security** - All JEPs must include a section describing any security risks and their severity.
-  The JEP must explain how the it proposes to deal with these risks.
-  If there are no security concerns, the section must say that.
+. **Security** - All JEPs must include a section ddescribing their impact
```
Comment (daniel-beck) [inline jep/1/README.adoc]: typo: ddescribing

File: jep/1/README.adoc [hunk 7]
```diff
@@ -553,15 +553,22 @@ community and discuss important objections or concerns raised
...
+  on the security of affected components, and, if any, how these are dealt with.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: 'these' and 'any' refers to 'impact', I'm not sure that's grammatical. Not sure how else to phrase it. Alternative, countable, term for impact perhaps?

File: jep/1/README.adoc [hunk 8]
```diff
@@ -553,15 +553,22 @@ community and discuss important objections or concerns raised
...
+  If the JEP has no impact on security, the section must say that.
 . **Infrastructure Requirements** -
-  All JEPs must include a section describing any infrastructure changes or addtions they require.
-  The JEP must explain the scope of these changes in reasonable detail.
-  If there are no infrastructure changes, the section must say that.
+  All JEPs must include a section describing their impact on
+  Jenkins project infrastructure,
+  including additions or changes, interactions with exiting components,
+  potential instabilities, service-level agreements,
+  and assigning responsibilty for continuing maintenance.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: typo: responsibilty

File: jep/1/README.adoc [hunk 9]
```diff
@@ -553,15 +553,22 @@ community and discuss important objections or concerns raised
...
+  Each JEP must explain the scope of infrastructure change with sufficient detail
+  to allow their initial and on-going cost (in both time and money) can be estimated.
```
Comment (daniel-beck) [inline jep/1/README.adoc]: Grammar: Each JEP <-> their. Either all JEPs, or its.

File: jep/1/README.adoc (hunk 7 above)
Comment (bitwiseman) [reply jep/1/README.adoc]: @daniel-beck 
Dude, you wrote the thing.  I applied your suggested change. Now you tell me to change it again?  😕  Okay, I'm changing it again.

Comment (bitwiseman) [reply jep/1/README.adoc]: @daniel-beck This is why I wanted you to submit the PR.

**Comment by bitwiseman** [comment]
@daniel-beck All good now?

**Pull request merged**